src/Native/Runtime/module.cpp

```diff
@@ -1270,3 +1270,49 @@ UInt32 GenericInstanceDesc::DacSize(TADDR addr)
     return GenericInstanceDesc::GetSize(dummyDesc.GetFlags(), arity);
 }
 #endif // DACCESS_COMPILE
+
+/* static */
+ModuleInfo * ModuleInfo::Create(void * pHeaderStart, void * pHeaderEnd)
```

Comment:
I'll change it to ModuleManager. I'll leave ModuleInfoRow as-is since that name makes more sense than "ModuleManagerRow"